### PR TITLE
Fixed grammatical errors in README.md

### PR DESCRIPTION
**Pull request by Joe-Sin7h**
My PR doesn't contain any point from below as it's fixing the grammatical errors in `README.md`.
### Checklist for PRs

- [ ] Added `.. versionadded:: version`, `.. versionchanged:: version` or `.. deprecated:: version` to the docstrings for user facing changes (for methods/class descriptions, arguments and attributes)
- [ ] Created new or adapted existing unit tests
- [ ] Added myself alphabetically to `AUTHORS.rst` (optional)

### If the PR contains API changes (otherwise, you can delete this passage)

* New classes:
    - [ ] Added `self._id_attrs` and corresponding documentation
    - [ ] `__init__` accepts `**_kwargs`
    
* Added new shortcuts:
    - [ ] In `Chat` & `User` for all methods that accept `chat/user_id`
    - [ ] In `Message` for all methods that accept `chat_id` and `message_id`
    - [ ] For new `Message` shortcuts: Added `quote` argument if methods accepts `reply_to_message_id`
    - [ ] In `CallbackQuery` for all methods that accept either `chat_id` and `message_id` or `inline_message_id`
    
* If relevant:

    - [ ] Added new constants at `telegram.constants` and shortcuts to them as class variables
    - [ ] Added new handlers for new update types
    - [ ] Added new filters for new message (sub)types
    - [ ] Added or updated documentation for the changed class(es) and/or method(s)
    - [ ] Updated the Bot API version number in all places: `README.rst` and `README_RAW.rst` (including the badge), as well as `telegram.constants.BOT_API_VERSION`
    - [ ] Added logic for arbitrary callback data in `tg.ext.Bot` for new methods that either accept a `reply_markup` in some form or have a return type that is/contains `telegram.Message`

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -123,7 +123,7 @@ You can install or upgrade python-telegram-bot with:
 
     $ pip install python-telegram-bot --upgrade
 
-Or you can install from source with:
+Or you can install from the source with:
```
Comment (Bibo-Joshi) [inline README.rst:126]: please revert this line - "installing from source" is pretty much a fixed expression IMO

File: README.rst [hunk 2]
```diff
@@ -167,8 +167,8 @@ Learning by example
 
 We believe that the best way to learn this package is by example. Here
 are some examples for you to review. Even if it is not your approach for learning, please take a
-look at ``echobot.py``, it is the de facto base for most of the bots out there. Best of all,
-the code for these examples are released to the public domain, so you can start by grabbing the
+look at ``echobot.py``. Itis the de facto base for most of the bots out there. Best of all,
```
Comment (Bibo-Joshi) [inline README.rst:170]:
```suggestion
look at ``echobot.py``. It is the de facto base for most of the bots out there. Best of all,
```

File: README.rst [hunk 3]
```diff
@@ -231,7 +231,7 @@ Contributions of all sizes are welcome. Please review our `contribution guidelin
 ========
 Donating
 ========
-Occasionally we are asked if we accept donations to support the development. While we appreciate the thought, maintaining PTB is our hobby and we have almost no running costs for it. We therefore have nothing set up to accept donations. If you still want to donate, we kindly ask you to donate to another open source project/initiative of your choice instead.
+Occasionally we are asked if we accept donations to support the development. While we appreciate the thought, maintaining PTB is our hobby ,and we have almost no running costs for it. We therefore have nothing set up to accept donations. If you still want to donate, we kindly ask you to donate to another open source project/initiative of your choice instead.
```
Comment (Bibo-Joshi) [inline README.rst:234]:
```suggestion
Occasionally we are asked if we accept donations to support the development. While we appreciate the thought, maintaining PTB is our hobby, and we have almost no running costs for it. We therefore have nothing set up to accept donations. If you still want to donate, we kindly ask you to donate to another open source project/initiative of your choice instead.
```

**Review by Bibo-Joshi — CHANGES_REQUESTED** [review]

**Comment by Bibo-Joshi** [comment]
Thanks for the PR! I rebased to the doc-fixes branch where we gather documentation contributions. Please be sure to resolve the merge conflicts and to also apply the changes to `README_RAW.rst`

**Comment by Joe-Sin7h** [comment]
@Bibo-Joshi I resolved the conflict and made the required changes. I'll also make same changes into `README_RAW.rst`

**Comment by Joe-Sin7h** [comment]
@Bibo-Joshi I have made grammatical changes into `README_RAW.rst` but I haven't added any extra section which was present in `README.rst` like **Learning examples**.

**Review by Bibo-Joshi — CHANGES_REQUESTED** [review]
Thanks for the update! Just one nitpick left :)

**Comment by Joe-Sin7h** [comment]
@Bibo-Joshi Done 👍🏻

**Pull request merged**